node-repository/src/main/java/com/yahoo/vespa/hosted/provision/NodeRepository.java

```diff
@@ -470,8 +470,8 @@ public Node fail(String hostname, Agent agent, String reason) {
      * @return the node in its new state
      * @throws NoSuchNodeException if the node is not found
      */
-    public Node park(String hostname, Agent agent, String reason) {
-        return move(hostname, Node.State.parked, agent, Optional.of(reason));
+    public Node park(String hostname, boolean keepAllocation, Agent agent, String reason) {
```

Comment:
Keep the same order of arguments as `move()`

Reply:
Done